content/en/guides/configuration-glossary/configuration-build.md

```diff
@@ -593,6 +593,8 @@ export default {
 
 Then, when launching `nuxt build`, upload the content of `.nuxt/dist/client` directory to your CDN and voilà!
 
+In Nuxt 2.15+, changing the value of this property at runtime will override the configuration of an app that has already been built.
```

Comment:
should this be a warning/info or best left as is??

Reply:
I'm happy either way. I didn't see any other `<alert>` through the docs so removed my initial info alert...